pkgs/applications/misc/swaynotificationcenter/default.nix

```diff
@@ -77,6 +77,7 @@ stdenv.mkDerivation (finalAttrs: rec {
   postPatch = ''
     chmod +x build-aux/meson/postinstall.py
     patchShebangs build-aux/meson/postinstall.py
+    sed "s#\"/usr/local/etc/xdg/swaync#\"$out/etc/xdg/swaync#" -i src/functions.vala
```

Comment:
Should probably use the dependency-less`substituteInPlace`, we don't even have explicitly dependency on gnused.

Reply:
> we don't even have explicitly dependency on gnused

Not strictly true: https://nixos.org/manual/nixpkgs/unstable/#sec-tools-of-stdenv

But yes, `substituteInPlace` is preferable for its clarity

Reply:
done